### PR TITLE
Add post-Installation instructions to both Windows and MacOS

### PR DESCRIPTION
**Pull request by jamesbt365**

### CODE REVIEW

**Comment by InfinityGhost** [comment]
Resolve merge conflicts.

**Comment by jamesbt365** [comment]
Resolved.

File: OpenTabletDriver.Web/Views/Wiki/Install/Windows.cshtml [hunk 1]
```diff
@@ -78,4 +78,12 @@
     <small class="text-muted">
         Note: Only install WinUSB if your tablet requires it and <b>only</b> on the interface stated.
     </small>
+    </p>
```
Comment (InfinityGhost) [inline OpenTabletDriver.Web/Views/Wiki/Install/Windows.cshtml:81]: Dangling `</p>` closing tag

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

**Comment by jamesbt365** [comment]
superseded by #120

**Pull request closed without merging**